extra_tests/regression_fits/feature_scaling_81.json

```diff
@@ -51,45 +51,45 @@
   ],
   "stop_epoch": 1100,
   "best_epoch": 1099,
-  "erf_tr": 2.6529221534729004,
-  "erf_vl": 2.1242189407348633,
-  "chi2": 1.4108729362487793,
+  "erf_tr": 2.803447723388672,
+  "erf_vl": 2.2027039527893066,
+  "chi2": 1.5391007661819458,
```

Comment:
so the mapping has changed?

Reply:
Yes slightly, I think it's because now the x=1 is usually included in the scaled xgrid where before it was not

Reply:
and why is it needed? Its contribution should be 0.

Reply:
Because previously x=1 was not in the fktable xgrids so I could just use the largest x-grid point as the last step in the grid before x=1. With pineappl x=1 is there and it should remain at 1. 

Before this change the x=1 in pineappl would be the last step/interval before 1 after scaling.